src/Components/CustomInput/CustomInput.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+type CustomInputProps = {
+  type: string;
+  placeholder: string;
+  // eslint-disable-next-line no-unused-vars
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const CustomInput: React.FC<CustomInputProps> = ({
+  type,
+  placeholder,
+  onChange,
```

Comment:
Add more props, to ensure that your component will be fully integrated with input from package. 

(disabled, customStyles, label, etc...)

Your type prop can be more specified -> type: 'number' | 'text' | etc ......